st2common/st2common/hooks.py

```diff
@@ -13,16 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import datetime
+import urlparse
```

Comment:
Yeah we had tried and tests kept breaking. Eventually we claim out code to be py3 compliant but can't recommend it to be run with py3.